openedx/adg/common/util/env_utils.py

```diff
@@ -0,0 +1,12 @@
+"""
+All utility methods related to env variables
+"""
+
+import os
+
+
+def is_testing_environment():
+    """
+    Check if our environment is testing or not.
+    """
```

Comment:
Due to the testing pipeline, I have moved `openedx/adg/common/util/env_utils.py` to `openedx/adg/lms/utils/env_utils.py`. Can we move this module as well?

Note: Folder name is now `utils` instead of `util`